services/types.go

```diff
@@ -278,6 +286,15 @@ type ServicePlacement interface {
 	// SetIsSharded() sets IsSharded
 	SetIsSharded(v bool) ServicePlacement
 
+	// CutoverNanos returns the cutover time in nanoseconds.
+	CutoverNanos() int64
+
+	// SetCutoverNanos sets the cutover time in nanoseconds.
+	SetCutoverNanos(cutoverNanos int64) ServicePlacement
+
+	// Proto generates a placement proto.
+	Proto() (*placementproto.Placement, error)
```

Comment:
I'm not sure if it's a good idea to tie all these interfaces into all the specific proto definitions